codersdk/deployment.go

```diff
@@ -1570,6 +1582,26 @@ Write out the current server config as YAML to stdout.`,
 			Group:       &deploymentGroupNetworkingHTTP,
 			YAML:        "proxyHealthInterval",
 		},
+		{
+			Name:        "Default Quiet Hours Schedule",
+			Description: "The default daily cron schedule applied to users that haven't set a custom quiet hours schedule themselves. The quiet hours schedule determines when workspaces will be force stopped due to the template's max TTL, and will round the max TTL up to be within the user's quiet hours window (or default). The format is the same as the standard cron format, but the day-of-month, month and day-of-week must be *. Only one hour and minute can be specified (ranges or comma separated values are not supported).",
+			Flag:        "default-quiet-hours-schedule",
+			Env:         "CODER_QUIET_HOURS_DEFAULT_SCHEDULE",
+			Default:     "",
+			Value:       &c.UserQuietHoursSchedule.DefaultSchedule,
+			Group:       &deploymentGroupUserQuietHoursSchedule,
+			YAML:        "defaultQuietHoursSchedule",
+		},
+		{
+			Name:        "Quiet Hours Window Duration",
+			Description: "The duration of quiet hours windows when triggered by cron. Workspaces can only be stopped due to max TTL during this window. Must be at least 1 hour.",
+			Flag:        "quiet-hours-window-duration",
+			Env:         "CODER_QUIET_HOURS_WINDOW_DURATION",
+			Default:     (4 * time.Hour).String(),
+			Value:       &c.UserQuietHoursSchedule.WindowDuration,
+			Group:       &deploymentGroupUserQuietHoursSchedule,
+			YAML:        "quietHoursWindowDuration",
+		},
```

Comment:
I wonder if this would be better served as two cron schedules; one for the beginning of the quiet hour window and one for the end. The current implementation forces each quiet hour window be the same length, but I could see it being helpful to have longer windows on the weekends or specific days.

Reply:
Seems like that could be an iterative addition if required. 

If we end up supporting 2 cron schedules instead of this format, we could always convert this to the second cron schedule.

Reply:
I'm going to remove the duration for now since it's unused. It was used at one point early on in this PR to apply jitter to the max_deadline, but we are instead going to use it (in a future PR) to allow users to postpone the max_deadline by a given amount if they happen to be working when the quiet hours ticks over.